src/components/cl/hier/cl_hier.c

```diff
@@ -38,6 +38,17 @@ static ucc_config_field_t ucc_cl_hier_lib_config_table[] = {
      ucc_offsetof(ucc_cl_hier_lib_config_t, sbgp_tls[UCC_HIER_SBGP_NET]),
      UCC_CONFIG_TYPE_STRING_ARRAY},
 
+    {"FULL_SBGP_TLS", "ucp",
+     "TLS to be used for FULL subgroup.\n"
+     "FULL subgroup contains all processes of the team",
+     ucc_offsetof(ucc_cl_hier_lib_config_t, sbgp_tls[UCC_HIER_SBGP_FULL]),
+     UCC_CONFIG_TYPE_STRING_ARRAY},
+
+    {"A2AV_NODE_THRESH", "0",
```

Comment:
i think we should keep existing naming scheme from TLs, i.e. COLLNAME_ALGNAME_PARAMNAME
for example ALLTOALLV_SPLIT_NODE_THRESH